debs/generate_debian_package.sh

```diff
@@ -35,14 +37,18 @@ clean() {
     exit ${exit_code}
 }
 
+ctrl_c() {
+    clean 1
+}
+
 build_deb() {
     CONTAINER_NAME="$1"
     DOCKERFILE_PATH="$2"
 
     SOURCES_DIRECTORY="${CURRENT_PATH}/repository"
 
     # Download the sources
-    git clone ${SOURCE_REPOSITORY} -b ${BRANCH} ${SOURCES_DIRECTORY} --depth=1
+    git clone ${SOURCE_REPOSITORY} -b ${BRANCH} ${SOURCES_DIRECTORY} --depth=1 || clean 1
```

Comment:
`build_deb` must return 0 if it finishes successfully or return 1 if there's an error. Then, the main function must detect the return code and execute `clean 1` or `clean 0` depending on the return value of the function.